### DISCUSSION TITLE
Why GetLoginRequest don't provide info whether it was Rejected?

### DISCUSSION THREAD

**Discussion by okAndrew** [ory/hydra#2608] [Q&A] [answered]
I do want to know whether I still can use _login_challenge_ by response for GetLoginRequest. 
Would be good to have:
1. WasHandled=true for Rejected _login_challenge_
2. RequestedAt info in response so that we can know whether Accept/Consent/etc is not expired yet

**Comment by vinckr** [comment, marked as answer]
Is the `requestWasHandledResponse` what you are looking for?
https://www.ory.sh/hydra/docs/reference/api/#requestwashandledresponse

You can see an overview of every information you can retrieve about the client sending the request and the request itself with the API here: 
https://www.ory.sh/hydra/docs/reference/api/#get-a-login-request

**Reply by vinckr** [reply]
Hey @okAndrew ,
Just checking in if your question was answered or if there was still something unclear.
I will mark this as answered for now, but feel free to come back to the discussion any time.

Thanks a ton! 🐝
